lib/onc_certification_g10_test_kit/bulk_data_group_export_stu1.rb

```diff
@@ -265,6 +265,10 @@ class BulkDataGroupExportSTU1 < Inferno::TestGroup
           ['type', 'url'].each do |key|
             assert file.key?(key), "Output file did not contain \"#{key}\" as required"
           end
+
+          if config.options[:require_absolute_urls_in_output]
+            assert file['url'].match?(%r{\Ahttps?://}), 'URLs in output file were not absolute as required.'
```

Comment:
I think this error message should be something like:
```
"URLs in output file must be absolute, but found `#{file['url']}`"
```
Also, we've verified that the `url` key is present, but not that its value is a String, so it isn't safe to call `match` on it.

Reply:
Added `to_s` which is defined on each data type, so should be fine?